templates/show_post.html

```diff
@@ -81,22 +80,22 @@
 					div_innerHTML += "</div>\n";
 					div.innerHTML = div_innerHTML;
 					//div.innerHTML = "TEST 456"
-				}
+				};
 			};
-			url = "/sub/" + sid + "/post/" + pid + "/" + cid + "/postcomment/";
-			content = "message=" + msg;
+			let url = "/sub/" + sid + "/post/" + pid + "/" + cid + "/postcomment/";
+			let content = "message=" + msg;
+			xhttp.open("POST", url, true);
 			xhttp.setRequestHeader("Content-type","application/x-www-form-urlencoded");
-			xhttp.open("POST",url,true);
 			console.log("url is" + url);
-			//xhttp.send(content);
-
+			xhttp.send(content);
+			event.preventDefault();
```

Comment:
This is the magic sauce; by default the onsubmit event presumes that it is sending form data back and full refreshing immediately afterwards. This is old behavior that messes with our own `XMLHttpRequest`. To prevent this interferance, we call `event.preventDefault()` to stop the refresh, after we have sent our data.